executable-spec/src/Cardano/Ledger/Spec/STS/Chain/Transaction.hs

```diff
@@ -47,17 +49,25 @@ import qualified Cardano.Ledger.Spec.STS.Dummy.UTxO as UTxO
 
 data TRANSACTIONS hashAlgo
 
-
+-- | Environment of the TRANSACTION STS
 data Env hashAlgo =
   Env { currentSlot :: !Slot
+      , closedVotingPeriods :: !(Map (SIPHash hashAlgo) (VotingPeriod hashAlgo))
+      -- ^ Records the closed voting periods per SIP
       , updatesEnv :: !(Environment (UPDATES hashAlgo))
+      -- ^ Environment of the child STS (UPDATES)
```

Comment:
I wonder if we should add these comments, since it should be clear from the variable names and their types.